docs/misc/commands.md

```diff
@@ -0,0 +1,122 @@
+---
+sidebar-position: 7
+---
+# Commands
+Commands are very useful in vanilla Minecraft and even more so with mods. They are the quickest way to access and modify data.
```

Comment:
Commands are meant to perform some operation within the game. Modifying data is only one aspect.